pandas/tests/reshape/test_pivot.py

```diff
@@ -142,6 +142,29 @@ def test_pivot_table_dropna_categoricals(self, dropna):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_pivot_with_non_observable_dropna_civ(self, dropna):
+        # gh-21370
+        arr = [np.nan, 'low', 'high', 'low', np.nan]
+        df = pd.DataFrame(
+            {"In": pd.Categorical(arr,
```

Comment:
Move the angled bracket to the line above. Move the word `arr` to the next line and move the subsequent kwargs up to that line accordingly (so long as they fit)